src/control/cmd/daos_server/README.md

```diff
@@ -0,0 +1,67 @@
+# DAOS Server
+
+The DAOS Server can be run either as a foreground process to issue hardware
+provisioning commands or as a background process (binary name followed by the
+"start" subcommand) when it will listen for instructions from the [management
+tool](/src/control/cmd/dmg/README.md) and manage bring-up of the DAOS system.
+A DAOS Server instance will typically be run on every storage node to
+orchestrate the operation of the managed data plane processes.
+
+## Configuration
+
+A server configuration file which is passed on invokation contains details of
+global and per-io-server parameters related to hardware devices and
+environments.
+
+## Certificate
+
+If the system is set up in normal mode (not insecure mode), the server will have
+been configured with a certificate with the CommonName "server" to identify
+itself and attest the validity of user credentials.
+
+For more details on how certificates are used within DAOS, see the
+[Security documentation](/src/control/security/README.md#certificate-usage-in-daos).
+
+## Data-Plane Communications
+
+I/O Server processes communicate with the DAOS Server using UNIX Domain Sockets
+set up on the storage node.
+
+### UNIX Domain Socket
+
+The default directory used for the UNIX Domain Sockets is
+`/var/run/daos_server`. Alternately, a directory may be specified in the server
+configuration file. The directory must exist, and the user starting the server
+process must have write access to it in order for the server to start up
+successfully. The server opens the socket and proceeds to listen for
+communications from local I/O server processes.
+
+### dRPC
+
+The protocol used to communicate between the DAOS server and I/O server processes
+is [dRPC](/src/control/drpc/README.md). Control and I/O servers are both capable
+of acting as dRPC server or client.
+
+For details on how gRPC communications are secured and authenticated, see the
+[Security documentation](/src/control/security/README.md#host-authentication-with-certificates).
```

Comment:
This section on gRPC should be moved below to the management tool discussion, or up into the certificate section.